### PR TITLE
Bgrenon autoscaling graphs

### PR DESCRIPTION
**Pull request by bgrenon**
Preview:
https://neon-next-git-bgrenon-autoscaling-graphs-neondatabase.vercel.app/docs/guides/autoscaling-guide#monitor-autoscaling

### CODE REVIEW

**Review by sharnoff — APPROVED** [review]
lgtm!

File: content/docs/guides/autoscaling-guide.md [hunk 1]
```diff
@@ -68,4 +68,32 @@ To edit a compute endpoint:
...
 
-The `neon_utils` extension provides a `num_cpus()` function for monitoring how the _Autoscaling_ feature allocates compute resources in response to workload. For more information, see [The neon_utils extension](/docs/extensions/neon-utils).
+From the Neon Console, you can view how your CPU and compute hour usage scales over time (last hour, day, and week). From the **Branches** page, open the branch you want to inspect, then open the **Edit** modal for its compute endpoint.
+
+![autoscaling graph example](/docs/guides/autoscaling_graphs_sample.png)
+
+Some key points about this Autoscaling view:
+
+- Allocation refers to the vCPU and memory size provisioned to handle current demand; autoscaling automatically adjusts this allocation, increasing or decreasing the allocated vCPU and memory size in a step-wise fashion as demand fluctuates, within your minimum and maximum limits.
+- Your minimum and maximum limits are shown as solid white lines. This represents the allocation boundary: the size of your allocated vCPU/memory stays within this range so long as your compute endpoint remains active. It scales to zero after the defined period of inactivity.
```
Comment (duskpoet) [inline content/docs/guides/autoscaling-guide.md:78]: I would suggest to either refrain from noting specific colors or designate the theme

**Review by duskpoet — COMMENTED** [review]
I would advice to hold off with the images while we update the visual part
FYI @lpetkov

File: content/docs/guides/autoscaling-guide.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/guides/autoscaling-guide.md:78]: maybe "solid horizontal lines"?

**Review by danieltprice — APPROVED** [review]

File: content/docs/reference/glossary.md [hunk 2]
```diff
@@ -433,6 +433,10 @@ Write-ahead logs in a specific LSN range.
 
 The stream of data written to the Write-Ahead Log (WAL) during transactional processing.
 
+## Working set
+
+A subset of frequently accessed or recently used data and indexes that ideally reside in memory (RAM) for quick access, allowing for better performance. See [how to size your endpoints](/docs/manage/endpoints#how-to-size-your-compute-endpoint) to learn how to set your minimum compute to an adequate size to handle your working set.
```
Comment (danieltprice) [inline content/docs/reference/glossary.md:438]:
```suggestion
A subset of frequently accessed or recently used data and indexes that ideally reside in memory (RAM) for quick access, allowing for better performance. See [how to size your compute](/docs/manage/endpoints#how-to-size-your-compute) to learn how to set your minimum compute to an adequate size to handle your working set.
```

File: content/docs/guides/autoscaling-guide.md (hunk 1 above)
Comment (bgrenon) [reply content/docs/guides/autoscaling-guide.md:82]: Good call. done.

**Review by lpetkov — APPROVED** [review]

**Pull request merged**